crates/types/src/config/admin.rs

```diff
@@ -92,7 +98,9 @@ pub struct AdminOptions {
     ///
     /// To update existing clusters use the `restatectl` utility.
     #[cfg_attr(feature = "schemars", schemars(with = "String"))]
-    pub default_partition_replication: PartitionReplication,
+    #[serde_as(as = "Option<serde_with::PickFirst<(_, ReplicationPropertyFromTo)>>")]
+    #[deprecated]
```

Comment:
This is mainly to avoid using it accidentally in code before it's completely removed. It also helps to spot the places it's being referenced

Reply:
Makes sense. Can we add a note and since when it's deprecated via `deprecated(since = "1.3.0", note = "xyz should be used instead. Will be removed with 1.4.0")`?